### PR TITLE
withWidth, use constants instead of magic numbers

### PR DESCRIPTION
**Pull request by oliviertassinari**
The width constants seem not to be used in the aor codebase. Is this on purpose?

### CODE REVIEW

**Comment by fzaninotto** [comment]
You're absolutely right. But you need to confirm to *prettier* code formatting. May I suggest you turn eslint auto-formatting on save in your IDE?

**Comment by oliviertassinari** [comment]
@fzaninotto This change was done with the code editor of GitHub. I will replace all the occurrences.

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (fzaninotto) [inline .prettierrc]: Please revert the prettier-related changes, these should not be in this PR.

File: src/auth/withPermissionsFilteredChildren.js [hunk 2]
```diff
@@ -42,21 +42,19 @@ export default BaseComponent => {
             const { authClient, ...props } = this.props;
             return (
                 <BaseComponent {...props}>
-                    {children && Array.isArray(children) ? (
```
Comment (fzaninotto) [inline src/auth/withPermissionsFilteredChildren.js]: please revert this change, as well as all the changes due to prettier configuration change

File: src/mui/input/CheckboxGroupInput.js [hunk 3]
```diff
@@ -128,21 +128,17 @@ export class CheckboxGroupInputComponent extends Component {
             <Checkbox
                 key={get(choice, optionValue)}
                 checked={
-                    value ? (
```
Comment (fzaninotto) [inline src/mui/input/CheckboxGroupInput.js]: please revert changes in this file

File: .prettierrc (hunk 1 above)
Comment (oliviertassinari) [reply .prettierrc]: There is not package lock of the dependencies. Feel free to amand my PR. As having the write access on the repository, you have write access to my fork's branch.

Comment (oliviertassinari) [reply .prettierrc]: Also I expect changing any part of the package.json invalidating travis cache and making all consecutives build red.

Comment (fzaninotto) [reply .prettierrc]: Not sure I understand what you mean. Do you say you had to change the prettier configuration because the prettier version wasn't fixed in the package, and prettier updated their defaults in a new version?

If that's the case, you should just remove the `~` in the prettier version in the `package.json`.

**Comment by oliviertassinari** [comment]
I'm giving up. I have tried with `"prettier": "1.5.2",` the diff is even worse. I have no clue about the prettier version used to format the source code.

**Comment by fzaninotto** [comment]
It should be easier to do in the `next` branch now.
